trace/tracestate.go

```diff
@@ -179,17 +179,14 @@ func (ts TraceState) Insert(key, value string) (TraceState, error) {
 	}
```

Comment:
The `Insert` function documentation needs to be updated to describe the new behavior. It should not mention an error is returned when capacity is exceeded.

Reply:
This was resolved, but the original comment remains:

https://github.com/open-telemetry/opentelemetry-go/blob/d0068b24aa1a6162dc7efd306202a0f0f74ca8c5/trace/tracestate.go#L172-L174

Is there commit that didn't get pushed yet?